src/pages/namespace/Events/History/EventsList.tsx

```diff
@@ -0,0 +1,145 @@
+import { Dialog, DialogContent } from "~/design/Dialog";
+import { Dispatch, SetStateAction, useState } from "react";
+import { FiltersObj, useEvents } from "~/api/events/query/get";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from "~/design/Table";
+
+import { Card } from "~/design/Card";
+import { EventSchemaType } from "~/api/events/schema";
+import Filters from "./components/Filters";
+import NoResult from "./NoResult";
+import { Pagination } from "~/componentsNext/Pagination";
+import { Radio } from "lucide-react";
+import Row from "./Row";
+import SendEvent from "./SendEvent";
+import ViewEvent from "./ViewEvent";
+import { itemsPerPage } from ".";
+import { useTranslation } from "react-i18next";
+
+const EventsList = ({
+  filters,
+  setFilters,
+  offset,
+  setOffset,
+}: {
+  filters: FiltersObj;
+  setFilters: (filters: FiltersObj) => void;
+  offset: number;
+  setOffset: Dispatch<SetStateAction<number>>;
+}) => {
+  const { t } = useTranslation();
+  const [eventDialog, setEventDialog] = useState<EventSchemaType | null>();
+
+  const { data, isFetched } = useEvents({
+    limit: itemsPerPage,
+    offset,
+    filters,
+  });
+
+  const handleOpenChange = (state: boolean) => {
+    if (!state) {
+      setEventDialog(null);
+    }
+  };
+
+  const handleFilterChange = (filters: FiltersObj) => {
+    setFilters(filters);
+    setOffset(0);
+  };
+
+  const numberOfResults = data?.events?.pageInfo?.total ?? 0;
+  const noResults = isFetched && data?.events.results.length === 0;
+  const showPagination = numberOfResults > itemsPerPage;
+  const hasFilters = !!Object.keys(filters).length;
+
+  return (
+    <div className="flex grow flex-col gap-y-3 p-5">
+      <div className="flex flex-row justify-between">
+        <h3 className="flex items-center gap-x-2 pb-2 pt-1 font-bold">
+          <Radio className="h-5" />
+          {t("pages.events.history.title")} {eventDialog ? "open" : "close"}
```

Comment:
I this `{eventDialog ? "open" : "close"}` some old debug code you forgot to remove? 

I was thinking if we should remove the headlines because it kind of only repeats what the tab is showing. What do you think? The "send new event" button could go on the opposite site of the filter button
![CleanShot 2023-08-11 at 17 57 40@2x](https://github.com/direktiv/direktiv-ui/assets/121789579/46766ca8-d05e-4bd2-953b-9c3f360a6ce9)

Reply:
Very good catch, thanks. Debug output removed in 6eb9f631c8a686c25b79b05961354646677cfb90.

I agree about the duplicate headline and will attempt a fix in a separate commit.

Reply:
I removed the redundant headers as suggested in 0d4483357f7211fea8e40f82516a8eb7884866f7.

It can get a bit crowded in the filters area (but that is already the case even without the "send event" button). I marked the send-event into a primary button to make it stand out more. What do you think?

![Screenshot from 2023-08-14 17-37-28](https://github.com/direktiv/direktiv-ui/assets/129740294/943dbfd1-73b5-47ff-bf00-6d8787009c3c)
![Screenshot from 2023-08-14 17-37-16](https://github.com/direktiv/direktiv-ui/assets/129740294/f17afa74-cf9d-411a-b5d8-5f299100f510)

Reply:
Using a primary button is a very good idea. It now differs very well from the filter buttons